moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -261,82 +243,42 @@ servo:
     }
   }
 
-############### POSE TRACKING PARAMETERS #########################
-
-  windup_limit: {
-    type: double,
-    default_value: 0.05,
-    description: "Maximum value of error integral for all PID controllers"
-  }
-
-  x_proportional_gain: {
-    type: double,
-    default_value: 1.5,
-    description: "Proportional gain value for the controller in x direction"
-  }
-
-  y_proportional_gain: {
-    type: double,
-    default_value: 1.5,
-    description: "Proportional gain value for the controller in y direction"
-  }
-
-  z_proportional_gain: {
-    type: double,
-    default_value: 1.5,
-    description: "Proportional gain value for the controller in z direction"
-  }
 
-  x_integral_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Integral gain value for the controller in x direction"
-  }
+########################### POSE TRACKING #######################################
 
-  y_integral_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Integral gain value for the controller in y direction"
-  }
-
-  z_integral_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Integral gain value for the controller in z direction"
-  }
-
-  x_derivative_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Derivative gain value for the controller in x direction"
-  }
-
-  y_derivative_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Derivative gain value for the controller in y direction"
-  }
-
-  z_derivative_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Derivative gain value for the controller in z direction"
-  }
+  pose_tracking:
+    linear_tolerance: {
+        type: double,
+        default_value: 0.001,
+        description: "The allowable linear error when tracking a pose.",
+        validation: {
+          gt<>: 0.0
+        }
+    }
 
-  angular_proportional_gain: {
-    type: double,
-    default_value: 0.5,
-    description: "Proportional gain value for angular control"
-  }
+    angular_tolerance: {
+        type: double,
+        default_value: 0.01,
+        description: "The allowable angular error when tracking a pose.",
+        validation: {
+          gt<>: 0.0
+        }
+    }
 
-  angular_integral_gain: {
-    type: double,
-    default_value: 0.0,
-    description: "Integral gain value for angular control"
-  }
+    frequency: {
+      type: double,
+      default_value: 50,
+      description: "The frequency for the pose tracking loop",
+      validation: {
+        gt<>: 0
+      }
+    }
```

Comment:
This is something I myself had questioned.
What is a more meaningful parameter ?  Should `Servo` itself use `frequency` ?
Was there any reason for `Servo`  using `publish_period` rather than a frequency ? @AndyZe

Reply:
no, i don't care either way

Reply:
The 'frequency` and `timeout` parameters of pose tracking has been removed, since there is no longer a pose tracking thread, `Servo` responds to pose commands in the same way it handles twist and jointjog